src/pages/Profile.tsx

```diff
@@ -109,42 +150,38 @@ const Profile = () => {
               <KeyboardBackspaceIcon color="secondary" />
             </IconButton>
             <Box>
-              <Typography sx={styles.displayName}>
-                {user.displayName}
+              <Typography sx={styles.headerName}>
```

Comment:
Do you think we should keep the look of the display name in the header the same as it is in the bio area?

Reply:
I'll actually revert this because it'd be better to keep it the same, but idk if you see this but I feel like there's an uneven amount of space between the top and bottom borders of the header and the text